package/MDAnalysis/analysis/density.py

```diff
@@ -455,6 +436,63 @@ def _conclude(self):
         density.make_density()
         self.density = density
 
+    @staticmethod
```

Comment:
Can you make sure that this private method is actually showing up in the docs? This seems useful to have documented so that one knows what happens when the grid is set.

Reply:
Fixed in c2c94b7